### ISSUE TITLE
Tip #108

### ISSUE THREAD

**Issue by JamesconDerulo** [OldSchoolRSTips/OldSchoolRSTips#1]
Tip 108 overflows so the text overlaps with the UI at the bottom

**Comment by OldSchoolRSTips** [comment]
Yes, unfortunately I'll jot tips down sometimes without checking how they render on the page. Fortunately, it's as easy as re-wording and compressing it and it usually fits on the second try.
